x-pack/plugins/encrypted_saved_objects/server/saved_objects/index.ts

```diff
@@ -177,7 +177,7 @@ export function setupSavedObjects({
           }
         }
 
-        return { ...finder, find: () => encryptedFinder() };
+        return { find: () => encryptedFinder(), close: finder.close.bind(finder) };
```

Comment:
__note:__ TypeScript will catch if new methods are added to PIT class.

Reply:
Creating a new class that inherits from `PointInTimeFinder` that only overrides the `find` method would probably have been cleaner. But as far as I can see it's in the `-internal` package, so it's not exposed to plugins, so I guess this is the best way.

Reply:
Yeah, correct, it's an internal-only class unfortunately